### DISCUSSION TITLE
Is mkcert downloaded everytime the server starts?

### DISCUSSION THREAD

**Discussion by florianmatz** [liuweiGL/vite-plugin-mkcert#24] [Q&A] [answered]
Subject says it all: Does the plugin do the download and certification generation each time the dev server is started?

**Comment by liuweiGL** [comment, marked as answer]
No, mkcert is downloaded only if the local binary does not exist, certificate is generated after [hosts](https://github.com/liuweiGL/vite-plugin-mkcert#hosts) changes.

**Reply by florianmatz** [reply]
Thank you for the first reply. Going to include it today. Is it generally working under Windows? There was this strange issue (https://github.com/liuweiGL/vite-plugin-mkcert/issues/15), but I’m not use if his problem was basically something else…

We might also have some window developer machines…

**Comment by liuweiGL** [comment]
>  Going to include it today. Is it generally working under Windows?

The plugin just calls [mkcert](https://github.com/FiloSottile/mkcert) ant supports the same platform.



> There was this strange issue (https://github.com/liuweiGL/vite-plugin-mkcert/issues/15), but I’m not use if his problem was basically something else…

Sorry i can't understand what you mean, if the plugin is not working please open an issue with log.
